src/settings.rs

```diff
@@ -0,0 +1,13 @@
+use serde::{Deserialize, Serialize};
+
+// Describe the settings your policy expects when
+// loaded by the policy server.
+#[derive(Serialize, Deserialize, Default, Debug)]
+#[serde(default)]
+pub(crate) struct Settings {}
+
+impl kubewarden::settings::Validatable for Settings {
+    fn validate(&self) -> Result<(), String> {
```

Comment:
Doesn't need to return a Result, since no error is raised.

Reply:
Ok, but this is a trait implementation. Is this possible? Afaik, this will cause a compilation error.

Reply:
doh, true, resolving.